components/molecules/table/formatter/link.js

```diff
@@ -16,6 +16,9 @@ import { StyledLink } from '../../../'
 import styled from 'styled-components'
 
 export const TableLink = (path, key, value, account) => ({ row }) => {
+  let skipLink = false
+  if (account && ['admin', 'tenant'].includes(row.type.toLowerCase())) skipLink = true
```

Comment:
@Ali-sharafi-git Agree should be in portal
@Ali-sharafi-git Always put curly brakcets on conditionals